sros2/sros2/policy/schemas/policy.xsd

```diff
@@ -10,15 +10,32 @@
     <xs:element name="policy" type="Policy" />
     <xs:complexType name="Policy">
         <xs:sequence minOccurs="1" maxOccurs="1">
-            <xs:element name="profiles" type="Profiles" />
+            <xs:element name="contexts" type="Contexts" />
         </xs:sequence>
         <xs:attribute name="version" type="xs:string" use="required" />
     </xs:complexType>
 
+    <xs:complexType name="Contexts">
+        <xs:sequence minOccurs="1" maxOccurs="unbounded">
+            <xs:element name="context" type="Context" />
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="Context">
+        <xs:sequence minOccurs="1" maxOccurs="unbounded">
+            <xs:element name="profiles" type="Profiles" />
+        </xs:sequence>
+        <xs:attribute name="path" type="xs:string" use="required" />
+    </xs:complexType>
+
     <xs:complexType name="Profiles">
         <xs:sequence minOccurs="1" maxOccurs="unbounded">
             <xs:element name="profile" type="Profile" />
         </xs:sequence>
+        <!-- <xs:sequence minOccurs="0" maxOccurs="1">
+            <xs:element name="metadata" type="xsd:anyType" />
+        </xs:sequence>
+        <xs:attribute name="type" type="xs:string" use="required" /> -->
```

Comment:
Even if we don't get around *using* the metadata in the tool by foxy, it would be great to get the new tag in the policy schema (so that any change in the tool doesn't require us to bump the version of the schema just for the metadata tag)

Reply:
I added this back, but wasn't sure of how we want to full integrate the `type` attribute, so I switched it to `use="optional"`.